include/CppUTest/UtestMacros.h

```diff
@@ -200,6 +200,29 @@
 #define UNSIGNED_LONGS_EQUAL_LOCATION(expected, actual, text, file, line)\
   { UtestShell::getCurrent()->assertUnsignedLongsEqual((unsigned long)expected, (unsigned long)actual, text, file, line); }
 
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
Although, I should qualify the above a bit.

With the `#ifdef`, attempting to use an assert macro with the feature unsupported will produce an `undefined symbol LONGLONGS_EQUAL` or similar compiler error.

Without the `#ifdef`, attempting to use an assert macro with the feature unsupported will produce an `cannot convert 5 to cpputest_longlong for parameter expected in assertLongLongsEqual` or similar compiler error.

So I guess it's just a question of which error message is more obvious to the user (and of course different compilers will have different messages, of varying understandability).